test/it/uk/gov/hmrc/tariffclassificationfrontend/ReferCaseSpec.scala

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmrc.tariffclassificationfrontend
+
+import com.github.tomakehurst.wiremock.client.WireMock._
+import org.scalatest.mockito.MockitoSugar
+import play.api.libs.ws.WSResponse
+import play.api.test.Helpers._
+import uk.gov.hmrc.tariffclassificationfrontend.models.CaseStatus
+import uk.gov.tariffclassificationfrontend.utils.{CasePayloads, Cases, EventPayloads}
+
+
+class ReferCaseSpec extends IntegrationTest with MockitoSugar {
+
+  "Case Refer" should {
+    val caseWithStatusOPEN = CasePayloads.jsonOf(Cases.btiCaseExample.copy(status = CaseStatus.OPEN))
+    val event = EventPayloads.event
+
+    "return status 200" in {
+      // Given
+      givenAuthSuccess()
+      stubFor(get(urlEqualTo("/cases/1"))
+        .willReturn(aResponse()
+          .withStatus(OK)
+          .withBody(caseWithStatusOPEN))
+      )
+      stubFor(post(urlEqualTo("/cases/1/events"))
+        .willReturn(aResponse()
+          .withStatus(CREATED)
+          .withBody(event))
+      )
+
+      // When
+      val response: WSResponse = await(ws.url(s"http://localhost:$port/tariff-classification/cases/1/refer").get())
```

Comment:
You shouldnt need all the `http://localhost:$port/tariff-classification` just use `$backendRoot`

Reply:
didn't seen it! 👍